### ISSUE TITLE
Add not georeferenceable option to preparation stage

### ISSUE THREAD

**Issue by BNAndras** [ohmg-dev/OldInsuranceMaps#1]
https://oldinsurancemaps.net/resource/328 is an example of a sheet that isn't a map. Currently, this sheet would sit in the unprepared area, but that's a difference between a page not yet reviewed but can be prepared and one that is reviewed but can't be prepared. Setting a sheet as not georeferenceable would set it aside from the usual georeferencing workflow but still allow it to be tracked as part of a volume.

**Comment by mradamcox** [comment]
Good idea and good example. Currently the way to achieve something like this would be to edit the document metadata and

1. remove the "unprepared" keyword
2. uncheck the "metadata_only" setting

This, however, would likely remove it from the volume summary, so isn't a full answer to what you're looking for.

**Comment by BNAndras** [comment]
We'd also need to clarify that this is for items that can't be georeferenced at all. After the splitting process, some small detail sections may be too limited in detail to be feasibly georeferenced. They may show a building not easily identified (or not extant) for example. Those could still be georeferenced in the future (even if unlikely) so they shouldn't be covered by this particular proposal.

**Comment by BNAndras** [comment]
https://oldinsurancemaps.net/documents/1153 is an example of a split section that can't be georeferenced in its current state. I could unspilt the sheet, georeference it, and then trim this area off. Or I can mark it as not georeferenceable but then we'd need to extend that keyword so it can be applied to prepared sections and not just sheets. Either one seems workable to me.

**Comment by mradamcox** [comment]
Ah, you found the sliver. Yeah, not entirely sure what to do with it, luckily its such a small area...

To your original point with this ticket, I think there are two new designations needed:

1. "Not a map" for items like the end page indexes https://oldinsurancemaps.net/documents/1041
    (This is what this ticket is primarily about)
2. "Ungeoreferencable" for items that just can't be located, or like the one linked above, are too small to even hold a set of distributed control points.

**Comment by BNAndras** [comment]
That works for me.

**Comment by mradamcox** [comment]
A new Non-Map category has been added to the volume summary page now, though I'd like to keep this ticket open until an effort is made to add more explanatory text to the buttons/description in the prep section about what setting a document as non-map means (and how to undo it).

**Comment by mradamcox** [comment]
After recent discussion with @rchampine, this really is a feature that is important to have for large-scale georeferencing efforts, and I've finally truly added it with the addition of a new "skip this piece" action.

![Image](https://github.com/user-attachments/assets/039841e4-148f-4008-a114-67b9a25bdaf9)
-->

![Image](https://github.com/user-attachments/assets/922315e1-6dda-4b7f-b8d7-57e2a62cb500)

This accomplishes the desire to not "lose" these items, they are still visible and can be "unskipped" and brought back into the process if new information arises, but it provides a place to put them outside the "prepared" section, which also removes them from the completion percentage that is displayed in the map list tables:

![Image](https://github.com/user-attachments/assets/c3123f4d-de5d-499c-a556-daa700087646)
